examples/hybrid/callbacks.jl

```diff
@@ -150,6 +160,23 @@ function turb_conv_affect_filter!(integrator)
     ODE.u_modified!(integrator, false)
 end
 
+horizontal_integral_at_boundary(f, lev) = sum(
+    Spaces.level(f, lev) ./ Fields.dz_field(axes(Spaces.level(f, lev))) .* 2,
```

Comment:
Can you remind me why the `*2` is needed here?

Reply:
Because at the boundary faces, `sum` calculates the integral over dz_field/2.

Reply:
Ah, right, thanks!